templates/vue/src/components/NodeModal.vue

```diff
@@ -685,6 +720,13 @@ export default {
     submitNode() {
       this.formErrors = this.validateNode(this.nodeData)
       if (!this.formErrors.length) {
+        if (
+          this.node.mediaType === "video" &&
+          this.nodeType !== "h5p" &&
+          this.videoUrlYoutubeID === ""
```

Comment:
It would be more readable if you change this and other similar cases in this file to a computed value.